client/matchmaking/find-match.jsx

```diff
@@ -80,7 +80,8 @@ const Actions = styled.div`
   margin: 16px 24px;
 `
 
-const Overline = styled(SubheadingOld)`
+const Overline = styled.span`
```

Comment:
We already have an Overline style, so this shouldn't inherit from `subtitle1`. If it doesn't match the other Overline style then it shouldn't be called that either